endpoints/citrus-http/src/main/java/org/citrusframework/http/server/HttpServer.java

```diff
@@ -202,7 +252,7 @@ protected void startup() {
             handlers.addHandler(contextCollection);
 
             handlers.addHandler(new DefaultHandler());
-            handlers.addHandler(new RequestLogHandler());
+            // TODO: handlers.addHandler(new RequestLogHandler());
```

Comment:
this has something to do with logs, I think the difference is... (TBD; add log example)

Reply:
yes, I do not recall why this is being added here. So let's remove and see